storage/src/vespa/storage/distributor/pendingclusterstate.cpp

```diff
@@ -306,7 +306,7 @@ PendingClusterState::getSummary() const
 {
     return Summary(getPrevClusterStateBundleString(),
                    getNewClusterStateBundleString(),
-                   (_clock.getTimeInMicros().getTime() - _creationTimestamp));
+                   std::chrono::microseconds(vespalib::count_us(_clock.getSystemTime().time_since_epoch()) - _creationTimestamp));
```

Comment:
Is there any point in explicitly truncating this down to microseconds here when the underlying type passed and stored as `duration`?

Reply:
Hardly